scripts/build_arbor.sh

```diff
@@ -30,13 +31,29 @@ fi
 # remove old build files
 mkdir -p "$arb_build_path"
 
+# build modcc
+if [ "$ns_arb_modcc" == "ON" ]; then
+    mkdir -p "$modcc_build_path"
+
+    # build external modcc
+    msg "ARBOR: build modcc"
+    cd "$modcc_build_path"
+    cmake .. -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ >> "$out" 2>&1
```

Comment:
You could still build with the same compiler as for the main Arbor library. The key is to avoid compiling with the `-DARB_ARCH` set to `$ns_arb_arch`, and use `native` instead:

```bash
cmake .. -DARB_ARCH=native >> "$out" 2>&1
```

Reply:
Done.